coralproject/cay
# user counts

## What?

Added user counts (only to the Search Creator screen). User counts will come from Pillar eventually for saved searches.

## How to test?

on the Search Create screen, when you drag the filters, the user count should update above the list.

### BUG 🐞 
~~this PR uncovered a bug in the infinite scrolling logic. sometimes we update the list and sometimes we clear it. If the user count shows as 1 or something and there are several users showing in the list, the user count is right. I'm going to open a new issue and fix it separate from this PR.~~ #228 

### UPDATE TO THE UPDATE
issue where user list was not cleared out is fixed with latest commit.

### something else weird
the Clauses are not displaying correctly when a dimension is active. We'll have to look into that later.
